.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove this file.

We started adding any IDE specific files into `.gitignore`